www/class/centreonStatsModules.class.php

```diff
@@ -116,15 +116,18 @@ private function getModuleObjects(array $installedModules)
      */
     public function getModulesStatistics()
     {
-        $data = array();
+        $data = [];
         $moduleObjects = $this->getModuleObjects(
             $this->getInstalledModules()
         );
-
         if (is_array($moduleObjects)) {
             foreach ($moduleObjects as $moduleObject) {
-                $oModuleObject = new $moduleObject();
-                $data[] = $oModuleObject->getStats();
+                try {
+                    $oModuleObject = new $moduleObject();
+                    $data[] = $oModuleObject->getStats();
+                } catch (\Throwable $e) {
```

Comment:
You should catch `StatisticsException`

Reply:
@callapa let me know what do you think about it :)

Reply:
we don't raise exception, so I think we should leave it only to avoid inheritance problems

Reply:
then, let `Throwable` to avoid side effects if an other Exception than `StatisticsException` is thrown